API-proposed.yaml

```diff
@@ -3299,12 +3632,64 @@ components:
           format: int64
         style:
           $ref: "#/components/schemas/OutputElementStyle"
-        xValues:
+        xValuesDescription:
           type: array
```

Comment:
this is not correct, I think. It's not an array. The swagger documentation needs to be updated for that and the API regenerated.